examples/Atmosphere.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2014 United States Government as represented by the Administrator of the
+ * National Aeronautics and Space Administration. All Rights Reserved.
+ */
+
+requirejs(['../src/WorldWind',
+        './LayerManager'],
+    function (ww,
+              LayerManager) {
+        "use strict";
+        
+        WorldWind.Logger.setLoggingLevel(WorldWind.Logger.LEVEL_WARNING);
+
+        var wwd = new WorldWind.WorldWindow("canvasOne");
+
+        var layers = [
+            {layer: new WorldWind.BMNGLayer(), enabled: true},
+            {layer: new WorldWind.CompassLayer(), enabled: false},
+            {layer: new WorldWind.CoordinatesDisplayLayer(wwd), enabled: false},
+            {layer: new WorldWind.ViewControlsLayer(wwd), enabled: false}
+        ];
+
+        for (var l = 0; l < layers.length; l++) {
+            layers[l].layer.enabled = layers[l].enabled;
+            wwd.addLayer(layers[l].layer);
+        }
+
```

Comment:
It would be better if the example demonstrated usage of the default image. Please remove specification of the night image here.